### PR TITLE
fix the empty item list

### PR DESCRIPTION
**Pull request by lovvge**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-249/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-249/2/index.html

**Comment by szha** [comment]
Job PR-249/3 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-249/3/index.html

File: src/gluonts/shell/serve.py [hunk 1]
```diff
@@ -66,21 +66,22 @@ def __iter__(self):
...
         )
-
-    # list(zip(timings, item_lengths)
+        for idx, (duration, input_length) in enumerate(
+            zip(timings, item_lengths), start=1
+        ):
+            logger.info(
+                f"\t{idx} took -> {duration:.2f}s (len(target)=={input_length})."
+            )
+    else:
```
Comment (jaheba) [inline src/gluonts/shell/serve.py]: I think this message is a bit non expressive without context. Can we add more information here if there are no inference items?

**Comment by jaheba** [comment]
Test are failing because of an earlier issue (my bad).

**Comment by szha** [comment]
Job PR-249/5 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-249/5/index.html

**Pull request merged**